extensions/cql/standard/clause_4_terms_and_definitions.adoc

```diff
@@ -6,12 +6,35 @@ This document uses the terms defined in Sub-clause 5.3 of [OGC 06-121r9], which
 For the purposes of this document, the following additional terms and definitions apply in addition to the terms defined
 in <<OAFeat-1,OGC API - Features - Part 1: Core>>.
 
+[[collection-def]]
+==== collection
+an object that groups multiple resources in a single unit
```

Comment:
We should use the definition from Common Part 2:

> A body of resources that belong or are used together. An aggregate, set, or group of related resources. (OGC 20-024)

Reply:
Updated the definition using the text provided.